### ISSUE TITLE
Nuget includes are not seen

### ISSUE THREAD

**Issue by g-glowacki** [Caphyon/clang-power-tools#123]
When a project uses nuget package e.g boost or GSL the includes are not seen:
It's a bug in 2.0

```
x.cpp(1): error: 'boost/optional.hpp' file not found [clang-diagnostic-error]
#include <boost/optional.hpp>
         ^


xo.cpp(1): error: 'boost/asio/io_service.hpp' file not found [clang-diagnostic-error]
#include "boost/asio/io_service.hpp"

x.cpp(4): error: 'gsl/gsl_assert' file not found [clang-diagnostic-error]
#include <gsl/gsl_assert>
         ^
```

**Comment by g-glowacki** [comment]
The workaround for this is to add paths to add nuget pacckage "include" subdir as argument for  parameter -I to clang compiler flags.

**Comment by cpp-red-lion** [comment]
We never tested projects with `Nuget` dependencies.
We'll have to investigate to see how the work with MSBuild property-sheets and VS projects.

Thanks for the suggestion !

**Comment by cpp-red-lion** [comment]
This issue was fixed in `version 2.4` released today.
Thanks for you feedback 👍
